### PR TITLE
fix(integrations) Fix crash when adding a repository fails

### PR DESCRIPTION
**Pull request by markstory**
Fix usage of undefined variable so we can show an error indicator when saving a repository fails.

Fixes JAVASCRIPT-4FS

### CODE REVIEW

File: tests/js/spec/views/organizationIntegrations/integrationRepos.spec.jsx [hunk 1]
```diff
@@ -15,6 +15,92 @@ describe('IntegrationRepos', function() {
...
+      let addRepo = Client.addMockResponse({
+        url: `/organizations/${org.slug}/repos/`,
+        method: 'POST',
+        body: {
+          id: 9,
+          integrationId: '1',
+          name: 'getsentry/sentry',
+          url: 'https://github.com/getsentry/sentry',
+          provider: {
+            name: 'GitHub',
```
Comment (lauryndbrown) [inline tests/js/spec/views/organizationIntegrations/integrationRepos.spec.jsx]: So why do we use GitHub here and not the Example integration as we would in the django tests? Seems like this is a broad change that isn't github specific.

File: src/sentry/static/sentry/app/views/organizationIntegrations/integrationRepos.jsx [hunk 2]
```diff
@@ -126,7 +126,7 @@ export default class IntegrationRepos extends AsyncComponent {
       },
       error: errorData => {
         let text = errorData.responseJSON.errors
-          ? data.responseJSON.errors.__all__
```
Comment (lauryndbrown) [inline src/sentry/static/sentry/app/views/organizationIntegrations/integrationRepos.jsx]: Oh good this was fixed.

**Comment by lauryndbrown** [comment]
@MeredithAnya I'm just commenting so I can get some practice reading react. Please feel free to actually review the pr. :)

File: tests/js/spec/views/organizationIntegrations/integrationRepos.spec.jsx [hunk 3]
```diff
@@ -15,6 +15,92 @@ describe('IntegrationRepos', function() {
     const integration = TestStubs.GitHubIntegration();
     const routerContext = TestStubs.routerContext();
 
+    describe('successful save', async function() {
+      let addRepo = Client.addMockResponse({
+        url: `/organizations/${org.slug}/repos/`,
+        method: 'POST',
+        body: {
```
Comment (billyvg) [inline tests/js/spec/views/organizationIntegrations/integrationRepos.spec.jsx]: Can we use anything in `tests/js/fixtures`?

Comment (markstory) [reply tests/js/spec/views/organizationIntegrations/integrationRepos.spec.jsx]: The fixture in `tests/js/fixtures/repository.js` gets close. I'll use that.

File: tests/js/spec/views/organizationIntegrations/integrationRepos.spec.jsx (hunk 1 above)
Comment (markstory) [reply tests/js/spec/views/organizationIntegrations/integrationRepos.spec.jsx]: I was matching up with the `TestStubs.GitHubIntegration()` which is the test fixture for integration objects.

Comment (MeredithAnya) [reply tests/js/spec/views/organizationIntegrations/integrationRepos.spec.jsx]: @lauryndbrown there was already a `JiraIntegration` but needed one to test repository functionality so I add `GitHubIntegration`. We could potentially have a sample integration test stub but generalizing for all the integrations seems like more effort than it was worth at the time.

**Review by billyvg — APPROVED** [review]
Looks good!

**Pull request merged**